### PR TITLE
Do not give admins' exercises for reviewing

### PR DESCRIPTION
**Pull request by vilmakoo**
:camel:

### CODE REVIEW

File: app/models/peer_review.rb [hunk 1]
```diff
@@ -10,8 +10,11 @@ class PeerReview < ApplicationRecord
   validates :comment, presence: true
 
   def draw_exercise(assignment)
+    regular_users = User.where(administrator: false)
+
     assignment.exercises
               .where(status: :finished)
+              .where(:user_id.in?(regular_users)) # do not give random test exercises made by creators for students to review
```
Comment (nygrenh) [inline app/models/peer_review.rb]: Does this work?

Comment (vilmakoo) [reply app/models/peer_review.rb]: At least when I tried it.

Comment (nygrenh) [reply app/models/peer_review.rb]: How about something like: `.where(user: regular_users)`

Comment (vilmakoo) [reply app/models/peer_review.rb]: okay so well okay erm it probably doesn't work after all :D

Comment (vilmakoo) [reply app/models/peer_review.rb]: that seems to work, fixing it, brb

**Pull request merged**